server/container/core/src/main/java/org/apache/james/server/core/AutomaticallySentMailDetectorImpl.java

```diff
@@ -34,6 +34,7 @@
 import org.apache.james.mime4j.stream.BodyDescriptor;
 import org.apache.james.mime4j.stream.MimeConfig;
 import org.apache.mailet.Mail;
+import org.apache.mailet.base.AutomaticallySentMailDetector;
 
 public class AutomaticallySentMailDetectorImpl implements AutomaticallySentMailDetector {
```

Comment:
why don't we move that in server/mailets?

Reply:
It's imo more related to the way we run mailet, thus it should rather go in mailetcontainer projects and we should enforce the `server/mailets` to use the API

Reply:
I had hard time reading that message but I guess I finally got it.